src/daemon/rpmostreed-transaction.cxx

```diff
@@ -482,6 +486,10 @@ transaction_dispose (GObject *object)
   g_clear_object (&priv->invocation);
   g_clear_object (&priv->cancellable);
   g_clear_object (&priv->sysroot);
+  if (priv->server)
+    {
+      g_dbus_server_stop (priv->server);
+    }
```

Comment:
Minor: I don't think we need this. Isn't it stopped when we unref the object just under?

Reply:
I think it may not be because something else is holding a reference, and we're just doing an unref, not doing https://developer-old.gnome.org/gobject/2.68/gobject-The-Base-Object-Type.html#g-object-run-dispose or so?

Reply:
In practice we were definitely leaking the socket without this change.

Reply:
Ahh gotcha. Interesting.